### DISCUSSION TITLE
How to check scte marker present in TV hls url?

### DISCUSSION THREAD

**Discussion by kapilgpt06** [futzu/threefive#53] [Q&A] [answered]
I have a tv HLS URL and I want to check scte-35 marker is present or not?
I tried through tsduck lib but I can't find any solution there.
Please share any command or solution
Thanks!

**Comment by futzu** [comment, marked as answer]
Sorry man, I am just now seeing this, 
use m3ufu.
https://github.com/futzu/m3ufu

**Reply by futzu** [reply]
That is exactly why I wrote m3ufu, and its built with threefive. 
I am so sorry have not responded sooner.
